tests/test_iter.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) QuantCo 2023-2024
+# SPDX-License-Identifier: BSD-3-Clause
+
+import pytest
+
+import ndonnx as ndx
+
+
+def test_iter_for_loop():
+    n = 5
+    a = ndx.array(shape=(n,), dtype=ndx.int64)
```

Comment:
```suggestion
    a = ndx.array(shape=(5,), dtype=ndx.int64)
```

Reply:
We use `n` in the assert in the loop below, too.